sdk/core/azure-core-experimental/src/main/java/com/azure/core/experimental/jsonpatch/JsonPatchDocument.java

```diff
@@ -22,15 +25,44 @@
     private static final ObjectMapper MAPPER = ((JacksonAdapter) JacksonAdapter.createDefaultSerializerAdapter())
         .serializer();
 
+    private static final String ADD = "add";
+    private static final String REMOVE = "remove";
+    private static final String REPLACE = "replace";
+    private static final String MOVE = "move";
+    private static final String COPY = "copy";
+    private static final String TEST = "test";
+
     private final ClientLogger logger = new ClientLogger(JsonPatchDocument.class);
 
     private final List<JsonPatchOperation> operations;
+    private final JsonSerializer serializer;
 
     /**
      * Creates a new JSON Patch document.
      */
     public JsonPatchDocument() {
+        this(null);
+    }
+
+    /**
+     * Creates a new JSON Patch document.
+     * <p>
+     * If {@code serializer} isn't specified {@link JacksonAdapter} will be used.
+     *
+     * @param serializer The {@link JsonSerializer} that will be used to serialize patch operation values.
+     */
+    public JsonPatchDocument(JsonSerializer serializer) {
         this.operations = new ArrayList<>();
+        this.serializer = serializer;
+    }
+
+    /**
+     * Gets the JSON Patch operations in this document.
+     *
+     * @return The JSON Patch operations in this document.
+     */
+    public List<Object> getJsonPatchOperations() {
```

Comment:
Can this return a `List<JsonPatchOperation>` instead?

Reply:
Done